src/App.test.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
+import './setup-tests';
```

Comment:
You shouldn't need to import this file in every test file. Jest should use this configuration by default.

Reply:
Removed these imports from the test files